server/routes/private.js

```diff
@@ -48,10 +48,21 @@ router.post('/book-appointment', async (req, res) => {
 
     if (!_.isNil(error)) res.send(error);
 
-    // Attempts to insert the appointment into the database.
-    const isSuccessfullyInserted = await appointmentHandler.bookAppointment(studentId, workerTimeslotId, purpose, studentNotes, workerComments);
+    // Checks worker timeslot status, either available or unavailable)
 
-    res.send(isSuccessfullyInserted);
+    const workerIsAvailable = await workerTimeslotHandler.checkWorkerAvailability(workerTimeslotId);
+
+    if(workerIsAvailable = true){
```

Comment:
Hi Nancy,
I would recommend using two equal signs for checking equality as one equal sign normally denotes assignment of a value to a variable. Additionally, since workerIsAvailable is a boolean value, it is possible to just check the value, e.g., `if (workerIsAvailable)`.

Reply:
Hi Amy,
Thank you for your suggestion!
My Action: I have changed line 55 to be `if(workerIsAvailable == true){`.

You can see this change in line 55 of server/routes/private.js

Please confirm if this takes care of your suggestion!

Reply:
Hi Nancy,
Yep, that works. Thank you!